contexts/MembershipContext/src/Domain/Model/Application.php

```diff
@@ -42,8 +41,9 @@ class Application {
 	private $isCancelled;
 	private $isConfirmed;
 	private $isDeleted;
+	private $donationReceipt;
 
-	public static function newApplication( string $type, Applicant $applicant, Payment $payment ): self {
+	public static function newApplication( string $type, Applicant $applicant, Payment $payment, ?bool $donationReceipt ): self {
```

Comment:
Needing a new parameter here makes me a bit sad, though can't think of a better way of doing this.